### PR TITLE
Add PW-RBSC-cor_6th-ed

### PR DESCRIPTION
**Pull request by UlrikeS91**
After several discussion with both @lzehl and @aeidi89, I manage to populate the content for Paxinos and Watson's 6th edition of their atlas. Since it's actually 3 atlases in one book (one for each anatomical plane) and they did not define a negative axis for the mediolateral axis (positive values in both directions), this is only the PR for the coronal atlas and includes 4 different CCSVs and corrsponding BAVs (2 different origins and left & right oriented because of the poorly defined mediolateral axis).

I also made a repository with an overview, the script and the input files here: https://github.com/UlrikeS91/AtlasIntegration-into-openMINDS/tree/main/PaxinosWatson-RBSC. 

As for the PR for Swanson, @aeidi89 could you please review the content one last time and @lzehl could you please review the rest?

### CODE REVIEW

File: instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Bregma-left.jsonld [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/PW-RBSC-corT_v2004-Bregma-left",
+  "@type": "https://openminds.ebrains.eu/core/CommonCoordinateSpaceVersion",
+  "abbreviation": "PW-RBSC-corT",
+  "accessibilty": {
```
Comment (aeidi89) [inline instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Bregma-left.jsonld:8]: GitHub Actions flagged this one as a spelling error

File: instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Bregma-right.jsonld [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/PW-RBSC-corT_v2004-Bregma-right",
+  "@type": "https://openminds.ebrains.eu/core/CommonCoordinateSpaceVersion",
+  "abbreviation": "PW-RBSC-corT",
+  "accessibilty": {
```
Comment (aeidi89) [inline instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Bregma-right.jsonld:8]: Spelling error?

File: instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Bregma-left.jsonld (hunk 1 above)
Comment (UlrikeS91) [reply instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Bregma-left.jsonld:8]: jup...should be "accessibility" (last i is missing...)

File: instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Bregma-right.jsonld (hunk 2 above)
Comment (UlrikeS91) [reply instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Bregma-right.jsonld]: Thanks! I corrected them now 🙂

File: instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Interaural-left.jsonld [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/PW-RBSC-corT_v2004-Interaural-left",
+  "@type": "https://openminds.ebrains.eu/core/CommonCoordinateSpaceVersion",
+  "abbreviation": "PW-RBSC-corT",
+  "accessibilty": {
```
Comment (aeidi89) [inline instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Interaural-left.jsonld]: Spelling error?

File: instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Interaural-right.jsonld [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpaceVersion/PW-RBSC-corT_v2004-Interaural-right",
+  "@type": "https://openminds.ebrains.eu/core/CommonCoordinateSpaceVersion",
+  "abbreviation": "PW-RBSC-corT",
+  "accessibilty": {
```
Comment (aeidi89) [inline instances/latest/commonCoordinateSpaceVersions/PW-RBSC-corT/PW-RBSC-corT_v2004-Interaural-right.jsonld]: Spelling error?

**Comment by UlrikeS91** [comment]
We found that my script introduced a typo in the type (instead of "sands", the path had "core"). The last commits fixed this.

**Pull request merged**